### PR TITLE
Better view on cart detail website

### PR DESCRIPTION
**Pull request by michalkrzem**
I change the way the cart details page is displayed. Only the visual aspects of the site, not the functionality.

image

### CODE REVIEW

**Comment by memorisanka** [comment]
Looks good for me. But I'm not sure about word "Product" in front of the name. Is it necessary? I think the name itself will be enough.

File: Dshop/apps/products_catalogue/templates/products_catalogue/cart_detail.html [hunk 1]
```diff
@@ -4,37 +4,47 @@
...
-                </form>
-            </div>
-            <div class="cart mt-4 align-items-center">
-                <form action="{% url 'delete_cart_item' item.product.slug item.id %}" method="post">
-                   {% csrf_token %}
-                   <input class="btn btn-danger text-uppercase mr-2 px-4" type="submit" value="DELETE">
-                </form>
+    <div class="card mb-5">
+        <div class="card-body">
+            <h5 class="card-title">Produkt: {{ item.product }}</h5>
```
Comment (jacoor) [inline Dshop/apps/products_catalogue/templates/products_catalogue/cart_detail.html]:
```suggestion
            <h5 class="card-title">{{ item.product }}</h5>
```

File: Dshop/apps/products_catalogue/templates/products_catalogue/cart_detail.html [hunk 2]
```diff
@@ -4,37 +4,47 @@
...
+            <p class="card-text">Cena jednostkowa: {{ item.price }}</p>
```
Comment (jacoor) [inline Dshop/apps/products_catalogue/templates/products_catalogue/cart_detail.html]:
```suggestion
            <p class="card-text">Cena: {{ item.price }}</p>
```

File: Dshop/apps/products_catalogue/templates/products_catalogue/cart_detail.html [hunk 3]
```diff
@@ -4,37 +4,47 @@
...
+            <p class="card-text">Całkowity koszt: {{ item.subtotal }}</p>
```
Comment (jacoor) [inline Dshop/apps/products_catalogue/templates/products_catalogue/cart_detail.html]:
```suggestion
            <p class="card-text">Suma: {{ item.subtotal }}</p>
```

**Review by jacoor — DISMISSED** [review]
Yes, i agree with @memorisanka that naming should be changed. Not critical, thus approve.

**Review by jacoor — APPROVED** [review]

**Pull request merged**